### PR TITLE
Branch for testing threads

### PR DESCRIPTION
**Pull request by AntonPanchishko**

### CODE REVIEW

File: src/main/java/Main.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+public class Main {
+    public static void main(String[] args) {
+        Counter counterForRunnable = new Counter();
+        Counter counterForThread = new Counter();
```
Comment (Max-Pochepets) [inline src/main/java/Main.java]: You should have one Counter for both threads

File: src/main/java/Counter.java [hunk 2]
```diff
@@ -0,0 +1,11 @@
+public class Counter {
+    private int count;
+
+    public int getCount() {
+        return count++;
+    }
```
Comment (Max-Pochepets) [inline src/main/java/Counter.java]: Let's leave `getCount()` as a normal getter and make a separate `increment()` method to increase count value

File: src/main/java/RunnableClass.java [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    public RunnableClass(Counter counter) {
+        this.counter = counter;
+    }
+
+    @Override
+    public void run() {
+        logger.info("Runnable " + Thread.currentThread().getName() + " was started ");
+        for (int i = 0; i <= COUNT_LENGTH; i++) {
+            logger.info(Thread.currentThread().getName() + " | " + counter.getCount());
+        }
```
Comment (Max-Pochepets) [inline src/main/java/RunnableClass.java]: You have Counter. What for do you need for loop?

**Review by Max-Pochepets — CHANGES_REQUESTED** [review]

**Review by Max-Pochepets — APPROVED** [review]

File: src/main/java/Main.java [hunk 4]
```diff
@@ -0,0 +1,9 @@
+public class Main {
+    public static void main(String[] args) {
+        Counter counter = new Counter();
+        RunnableClass runnableClass = new RunnableClass(counter);
+        ThreadClass threadClass = new ThreadClass(counter);
+        new Thread(runnableClass).start();
+        threadClass.run();
```
Comment (Sofasmile) [inline src/main/java/Main.java]: what is the difference between start() and run()?

Comment (AntonPanchishko) [reply src/main/java/Main.java]: As I understand, start() we use for classes which immplement interface Runnable 
and run() we use for classes which extend Thread

Comment (Sofasmile) [reply src/main/java/Main.java]: не зовсім так
start() запускає виконання в новому потоці, а run() запускає виконання в поточному потоці, тобто в тому потоці що main

**Review by Sofasmile — APPROVED** [review]

**Pull request merged**